stl/inc/atomic

```diff
@@ -1148,7 +1148,7 @@ struct _Atomic_storage<_Ty&, 16> { // lock-free using 16-byte intrinsics
 
     _Atomic_storage() = default;
 
-    /* implicit */ constexpr _Atomic_storage(conditional_t<is_reference_v<_Ty&>, _Ty&, const _TVal> _Value) noexcept
+    /* implicit */ constexpr _Atomic_storage(conditional_t<is_reference_v<_Ty&>, _Ty&, const _TVal&> _Value) noexcept

```

Comment:
`is_reference_v<_Ty&>` was intended, see #843.

Reply:
It took me a while to remember what was going on here, but it's the TRANSITION comment a few lines above.